File: Python2/1001.py
#1001	Extremamente Básico	Accepted	Python	0.012	29/09/2017 13:05:38
A = None
B = None

def read_integer():
  try:
    # read for Python 2.x
    return int(raw_input())
  except NameError:
    # read for Python 3.x
    return int(input())


A = read_integer()
B = read_integer()
print(str("X = ") + str(A + B))

File: JavaScript/1001.js
//1001	Extremamente Básico	Accepted	JavaScript	0.056	28/07/2019 15:43:14
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');
    
var a = parseInt(lines.shift());
var b = parseInt(lines.shift());
   
console.log('X = ' + (a+b));